api/lib/domain/models/Course.js

```diff
@@ -17,6 +20,37 @@ class Course {
     // transformed
     this.nbChallenges = this.challenges.length;
```

Comment:
On pourrait p-e changer ça en computed property.
`
get nbChallenges() { return this.challenges.length }
`

Reply:
ok